examples/zephyr/golioth_basics/prj.conf

```diff
@@ -12,17 +16,6 @@ CONFIG_NET_SHELL=y
 # Application
 CONFIG_MAIN_STACK_SIZE=4096
 
-# Golioth specific stuff
-CONFIG_MBEDTLS=y
-CONFIG_MBEDTLS_DTLS=y
-CONFIG_NET_SOCKETS=y
-CONFIG_NET_UDP=y
-
-# Optional for some targets
-CONFIG_DNS_RESOLVER=y
-CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
-CONFIG_NET_SOCKETS_ENABLE_DTLS=y
-
 # TLS configuration
 CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_HEAP_SIZE=10240
```

Comment:
Some of these TLS options are enabled by `GOLIOTH_AUTH_PSK_MBEDTLS_DEPS` in `port/zephyr/Kconfig` so we don't need to set them here.